src/app/components/hris/employees/employee-profile/accordions/accordion-profile/accordion-profile-employee-details/accordion-profile-employee-details.component.html

```diff
@@ -65,9 +65,11 @@
                   <mat-form-field appearance="outline" style="width: 100%;">
                     <mat-label>Initials</mat-label>
                     <input matInput formControlName="initials">
-                    <mat-error
-                      *ngIf="sharedAccordionFunctionality.employeeDetailsForm.controls['initials'].hasError('pattern') && !sharedAccordionFunctionality.employeeDetailsForm.controls['initials'].hasError('required')">
-                      Initials must not contain numbers
+                    <mat-error *ngIf="sharedAccordionFunctionality.employeeDetailsForm.controls['initials'].hasError('initialsNoNumbers')">
+                      Initials must not contain numbers.
+                    </mat-error>
+                    <mat-error *ngIf="sharedAccordionFunctionality.employeeDetailsForm.controls['initials'].hasError('initialsUppercase') && !sharedAccordionFunctionality.employeeDetailsForm.controls['initials'].hasError('initialsNoNumbers')">
+                      Initials must be in uppercase.
```

Comment:
Fix: rather remove this validation to allow the user to use small case but when it gets saved it stores it as uppercase. i.e input is br or Br or BR or bR etc. but always saved and returned as "BR"